common/djangoapps/student/forms.py

```diff
@@ -80,8 +82,14 @@ def save(
             # Email subject *must not* contain newlines
             subject = subject.replace('\n', '')
             email = loader.render_to_string(email_template_name, context)
-            password_reset_link = loader.render_to_string('emails/password_reset_email_link.txt', context)
-            MandrillClient().send_password_reset_email(user.email, {
+
+            password_reset_link = context['protocol'] +\
+                '://' +\
+                site_name +\
+                reverse('student.views.password_reset_confirm_wrapper',
+                        kwargs={ 'uidb36': int_to_base36(user.id), 'token': context['token'] })
```

Comment:
Please read about the multiple ways of string concatenation in python and which is batter. In my opinion you should use "{protocol}://{site_name}{path}".format(protocol=context['protocol'], site_name=site_name, path=path)

Reply:
done.